src/__experimental__/components/date/date.spec.js

```diff
@@ -712,6 +713,17 @@ describe('when the calendar icon is clicked', () => {
   });
 });
 
+it('the required prop is passed to the input', () => {
+  const wrapper = render({ label: 'required', required: true });
+
+  const input = wrapper.find('input');
+  expect(input.at(0).prop('required')).toBe(true);
+  expect(input.at(1).prop('required')).toBe(undefined); // hidden input
+
+  const label = wrapper.find(Label);
+  expect(label.prop('isRequired')).toBe(true);
+});
```

Comment:
looks like it could be split into two separate unit tests

Reply:
Updated test description because the setup is the same.